Chaincase.UI/Pages/CoinJoinPage.razor

```diff
@@ -1,51 +1,142 @@
 ﻿@page "/coinjoin"
 @using Chaincase.UI.Services
+@using WalletWasabi.CoinJoin.Client.Rounds;
+@using Chaincase.Common
+@using Chaincase.UI.Components;
+
 @inject UIStateService UiStateService
 @inject NavigationManager NavigationManager
+@inject Global Global
+@inject CoinJoinViewModel CoinJoinViewModel
+
+@inherits ReactiveComponentBase<CoinJoinViewModel>
 
 <ion-content class="ion-padding">
-    <!-- CoinListTemplate -->
-    @if (IsQueuedToCoinJoin)
+
+    @if (!ViewModel.IsSynced)
     {
-        <p style="color: red;">
-            Chaincase must be open to complete a CoinJoin.
-            We'll notify you when registration ends to open.
-        </p>
+        <ion-text color="primary">
+            <h2>Once your wallet has sync'd, you can pick coins to mix!</h2>
+        </ion-text>
     }
-
-    <p>₿ @RequiredBTC required to CoinJoin</p>
-
-
-    @if (IsQueuedToCoinJoin)
+    else
     {
-        <IonButton href="/send/sent" expand="block">EXIT COINJOIN </IonButton>
+        @if (ViewModel.IsQueuedToCoinJoin)
+        {
+            <p>
+                🎉 Inputs have been registered
+            </p>
+            <p class="danger-text">
+                Chaincase must be open to complete a CoinJoin.
+                We'll notify you when registration ends to open.
+            </p>
+        }
 
-    } else
-    {
-        <IonButton href="/send/sent" expand="block">COINJOIN</IonButton>
+        <ion-text color="primary">
+            <h2>
+                Amount Selected:
```

Comment:
consider `Amount Queued`, as that's what's displayed

Reply:
I like `Amount Queued`! But it would need to be conditionalized to only show when they actually queue.

Reply:
oh I was confused about this. The label is "AmountQueued" but it's actually showing whats selected. I'm sure we could use the same amount no matter what and just show the status along with it

would be nice to show status by individual coin. the whole UI here is a disaster and always has been imo. time to innovate